packages/react-admin-mui/src/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,7 +1,7 @@
-import { Link } from "@material-ui/core";
+import { Grid, Link, Typography } from "@material-ui/core";
+import { KeyboardArrowRight as ArrowIcon } from "@material-ui/icons";
```

Comment:
Path-imports bei icons nehmen.

Reply:
tree shaking ist auch im dev env aktiviert

Reply:
Genau, wir haben tree shaking im dev genau wegen dem aktiviert. Der performance (build time) unterschied (vorallem bei icons) ist messbar aber nicht gravierend. Und das ergebnis ist sowiso gleich.

In der material-ui docu sind überall keine path-imports, und ich glaube wir machen es meistens auch nicht (korrigiert mich!)